ng/src/web/pages/portlists/component.js

```diff
@@ -66,16 +73,21 @@ class PortListComponent extends React.Component {
     if (entity) {
       gmp.portlist.get(entity).then(response => {
         const port_list = response.data;
+        this.created_port_ranges = [];
+        this.deleted_port_ranges = [];
         this.setState({
           comment: port_list.comment,
           port_list,
           name: port_list.name,
           portListDialogVisible: true,
+          tmp_port_ranges: port_list.port_ranges,
```

Comment:
I would also rename this variable to port_port ranges even if the are only temporary because this is the list with the visible port ranges which should be saved and used at the end.